### PR TITLE
new icon: cairo (original, plain, original-wordmark, plain-wordmark)

### LINKED ISSUE #1378: [ICON REQUEST]: Cairo graphics
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Cairo is a 2D graphics library with support for multiple output devices. 

### Links and sources

https://www.[cairographics.org](https://www.cairographics.org/)/

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by lunatic-fox**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1378**

## Link to prove your SVG is correct and up-to-date.
https://www.cairographics.org/cairo-banner.png

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -964,6 +964,28 @@
             }
         ]
     },
+    {
+        "name": "cairo",
+        "altnames": [],
```
Comment (Snailedlt) [inline devicon.json:969]:
```suggestion
        "altnames": ["cairographics"],
```

**Review by Snailedlt — DISMISSED** [review]
Up to you if you want to add the altname or not. Couldn't find it any other place than on their website, but I guess it makes searching easier anyways? I dunno, up to you

File: devicon.json (hunk 1 above)
Comment (lunatic-fox) [reply devicon.json:969]: Agreed and committing!

**Review by Snailedlt — APPROVED** [review]
LGTM! :heavy_check_mark:

**Comment by lunatic-fox** [comment]
I think I made a mistake on this one since Cairo graphics already have `.svg` files to work on:
- https://commons.wikimedia.org/wiki/File:Cairo_banner_1.svg
- https://en.wikipedia.org/wiki/File:Cairo_banner_closeup.svg

I made the icons based on a `.png` file of their logo.
https://www.cairographics.org/cairo-banner.png

And in those files the logo have a totally different color pattern.

In order to fix it I'll make an icon bringing their `.svg` files to their website current color pattern.
  `(pale blue scarabs) > (yellow scarabs)`

I think this is more accurate, because keep the icon original nodes. And because their `.svg` files are from 2005 and on their website they are using the style I've picked and the latest version of the software is from 2020.

Before my commit I wanna be sure if everything is ok.

**Comment by Snailedlt** [comment]
@lunatic-fox Sounds good to me. Feel free to go ahead with the change

**Review by Snailedlt — APPROVED** [review]
Looks good! ✔️ 
Liked the old one better personally, but this is more accurate to their current logo, so I think we should keep this one 💯

**Comment by Snailedlt** [comment]
@lunatic-fox 
Thanks for your contribution again, you've been on fire this past week 🔥

**Comment by lunatic-fox** [comment]
> Looks good! ✔️ Liked the old one better personally, but this is more accurate to their current logo, so I think we should keep this one 💯

Me too! 😅

File: devicon.json [hunk 2]
```diff
@@ -964,6 +964,28 @@
             }
         ]
     },
+    {
+        "name": "cairo",
+        "altnames": ["cairographics"],
+        "tags": [
+            "graphic",
+            "library"
```
Comment (Panquesito7) [inline devicon.json:972]:
```suggestion
            "library",
            "c",
            "open-source"
```

Comment (lunatic-fox) [reply devicon.json:972]: Agreed! ✔
Committing changes!

**Review by Panquesito7 — APPROVED** [review]
Awesome work. Thanks! 🚀

**Pull request merged**